Readability.js

```diff
@@ -721,11 +721,11 @@ Readability.prototype = {
           }
         }
 
-        // Remove empty DIV, SECTION, and HEADER nodes
+        // Remove DIV, SECTION, and HEADER nodes without any content(e.g. text, image, video, or iframe).
```

Comment:
Hm. Without any CSS, an `hr` and `br` have an actual visual effect on the page, though, so we're removing content that might be important to keep the article readable in some cases. If we detect containers with only an hr/br in them, can we reparent the hr/br to the parent of the container and then remove the container, or does that still not help? I guess it might not because I'm not sure the sibling joining logic will kick in at that point...

This is annoying because really, the `hr` and/or `br` belong inside the section that they start/end (depending on the semantics), but that's not where they are in the markup. :-(

If we can't come up with something better I guess we should take this, even if it removes these elements.

Out of curiosity, in pages where this happens, are the hr/br styled to no longer affect the visual appearance of the page? I wonder why they're included at all...